lib/src/main/java/uk/gov/di/ipv/core/library/persistence/item/CriResponseItem.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.di.ipv.core.library.persistence.item;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbSortKey;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+import java.time.Instant;
+
+@DynamoDbBean
+@ExcludeFromGeneratedCoverageReport
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class CriResponseItem implements DynamodbItem {
+    private String userId;
+    private String credentialIssuer;
+    private String credential;
```

Comment:
Maybe rename `credential` to `issuerResponse` or equivalent. This is the response from the issuer userinfo endpoint